src/stories/Library/material-search/material-search.scss

```diff
@@ -0,0 +1,235 @@
+// Variables
+$_gin_xs_spacing: 0.5rem;
+$_gin_m_spacing: 1rem;
+$_gin_border_color: #8e929c;
+$_gin_font_weight_semibold: 525;
+
+$_color__background-light: #eeeeed;
+$_color__hover-background: #8e929c;
+
+$_box-shadow1: 1px 1px 5px 1px rgba(0, 0, 0, 0.1);
+$_box-shadow: 0px 0px 0px 2px rgba(0, 0, 0, 0.25);
+$_loading-spinner-size: 40px;
+
+// These heights are explicitly set to covers used in the material search.
+// The reason for this is, that the covers must all use "large" as their size
+// prop, because this prop is used to determine which size of image is
+// displayed in the cover as provided by cover-service.
+// Example: A small "size" prop is used on the listResults and preview in
+// material search, but the component being used for displaying the materials
+// (I.e RecommendedMaterial) has its' prop set to "large", then image will
+// potentially not be displayed at all, since a "large" version is not served
+// by cover-service, but a small version is.
+
+// Arguably, this is a flaw in cover service, and should be fixed there, but
+// since there is no solution to this, and it potentially could happen for the
+// vast majority of materials I have decided to manually overwrite this here,
+// to ensure that the covers are displayed correctly and avoid frustration.
+$MATERIAL_SMALL_MOBILE: 104px;
+$MATERIAL_SMALL_DESKTOP: 137px;
+
+$list_header_height: 50px;
+
+// Material Search
+.material-search {
+  max-width: 800px;
```

Comment:
Consider documenting the origin of these fixed values.

There are some fixed values here - 800px, 220px, 170px - where I wonder what the relationship is to the rest if the layout.

Reply:
Not sure what to document really, these could be different since they are just a decision i made. If the designer wants it to change i think we should just change it then. I prefer the max width since it keeps the design compact, the alternative is that the components can grow to 1280px(gin styled container) which i think looks silly 🤷